src/search.cpp

```diff
@@ -335,7 +335,11 @@ void Thread::search() {
 
   std::memset(ss-7, 0, 10 * sizeof(Stack));
   for (int i = 7; i > 0; i--)
-     (ss-i)->continuationHistory = &this->continuationHistory[NO_PIECE][0]; // Use as sentinel
+  {
+      (ss-i)->continuationHistory = &this->continuationHistory[0][NO_PIECE][0]; // Use as sentinel
+      (ss-i)->continuationHistory = &this->continuationHistory[1][NO_PIECE][0]; // Use as sentinel
+  {
```

Comment:
Echoing @vondele's concern, is this intended? It seems like you're assigning (ss-i)->continuationHistory twice.

Reply:
Not intended...I will remove the redundancy